tensorflow-keras/src/main/java/org/tensorflow/keras/utils/TypeUtils.java

```diff
@@ -0,0 +1,98 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=======================================================================*/
+package org.tensorflow.keras.utils;
+
+import org.tensorflow.DataType;
+import org.tensorflow.types.TBfloat16;
+import org.tensorflow.types.TBool;
+import org.tensorflow.types.TFloat16;
+import org.tensorflow.types.TFloat32;
+import org.tensorflow.types.TFloat64;
+import org.tensorflow.types.TInt32;
+import org.tensorflow.types.TInt64;
+import org.tensorflow.types.TString;
+import org.tensorflow.types.TUint8;
+
+
+/**
+ *
+ * @author Jim Clarke
+ */
+public class TypeUtils {
+    
+    //TODO
+    public static boolean isComplex(DataType dtype) {
```

Comment:
Do we need this if there is no support for it?

Reply:
I can remove it. It was just a placeholder for the future.

Reply:
I removed the functionality of TypeUtils.java to DataType.java. I have removed TypeUtils.java from this PR and will be checking in the changes to DataTypes and its instances (e.g. TFloat32). I did not include isComplex().